src/js/facility-locator/components/markers/TownHall.jsx

```diff
@@ -0,0 +1,30 @@
+import DivMarker from './DivMarker';
+import React, { Component, PropTypes } from 'react';
+
+class TownHall extends Component {
+  render() {
+    const style = {
+      backgroundColor: '#2e8540',
+      borderRadius: '50%',
+      marginTop: '-12px',
+      marginLeft: '-12px',
+      width: '24px',
+      height: '24px',
+      zIndex: '999',
+    };
```

Comment:
@carlmjohnson Yes. I'm on the fence about whether to have these map marker components completely self-contained, i.e. there is no way to screw with their styles outside of the component, especially since they are Map Markers and should never be used independently outside of a map.

That said this TownHall component was a test and will probably go away if DSVA decides to use Numbered Map Markers instead of Custom Icons.